src/dotnet/commands/dotnet-test/Program.cs

```diff
@@ -39,17 +39,17 @@ public static int Run(string[] args)
                 CommandOptionType.SingleValue);
 
             var listTestsOption = cmd.Option(
-                "-lt|--listTests",
+                "-t|--list-tests",
```

Comment:
@blackdwarf can this be `-l` for the verb "list" here?  I think `dotnet test -l` seems more intuitive here ("test" shows up in the verb and `-l` usually means list)